lib/index.js

```diff
@@ -93,6 +93,7 @@ class Flooper{
 
     if(blockX < -(blockComputedWidth + this.bufferSize) ) {
       block.style.order=this.order;
+      block.style["-ms-flex-order"]=this.order;
```

Comment:
Does this work mate? Even though we're not using dot notation (`style.property`), if you print the style object, you'll see that all properties are camelCased. I know that in chrome using `.style["marginRight"]` or `.style["margin-right"]` produces the same effect but not sure about other browsers

Searching on [browserStack](https://cl.ly/1b0J0l3k0m30), and  the property that on IE is `style.msFlexOrder`.

Anyway you've probably have tested this anyway so just FYI.

Btw did you knew that you couldn't change and tweak the dom with IE10 dev tools? jiz, web development on back in the days must have been harsh

Reply:
Well... I tried a couple of methods, and this one worked... `style.msFlexOrder` is probably more coherent with the other one.
IE10 dev tools? more like dumbifying fools... they are just a joke